content/review.rst

```diff
@@ -54,7 +57,7 @@ General
   This will help a lot with the review. Land your changes with the ugly
   names and then do a follow-up branch for the renaming.
 
-* Feel free to submit as many merge request as you can. The size of 
+* Feel free to submit as many merge request as you can. The size of
```

Comment:
Now that you've touched the line above, I think you should also correct the grammar error in it. :-]